lib/util.js

```diff
@@ -3,11 +3,15 @@ const zero = new BN(0)
 
 module.exports = {
 
-  stringifyBalance (balance, bnDecimals) {
+  stringifyBalance (balance, bnDecimals, formattingDecimals) {
     if (balance.eq(zero)) {
       return '0'
     }
 
+    if (!formattingDecimals) {
+      formattingDecimals = 5;
```

Comment:
Let's leave the default at 3, so that this is not a breaking change for any users of this library. We can then pass in 5 where we need to in metamask-extension